engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -384,9 +388,14 @@ class PlatformDispatcher {
     required ViewFocusState state,
     required ViewFocusDirection direction,
   }) {
```

Comment:
This is out-of-scope for this PR, but why is `requestViewFocusChange` on `PlatformDispatcher`? I believe we wanted to avoid APIs that accepted `viewId`s. Should we deprecate this member and add a `FlutterView.requestFocusChange` that accepts a `FlutterView`? cc @justinmc @goderbauer